src/main/java/com/sprint/mission/discodeit/exception/DiscodeitException.java

```diff
@@ -0,0 +1,16 @@
+package com.sprint.mission.discodeit.exception;
+
+import java.time.Instant;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public class DiscodeitException extends RuntimeException {
+
+  private final Instant timestamp;
+  private final ErrorCode errorCode;
+  private final String details;
+
+
+}
```

Comment:
```suggestion
@Getter
@RequiredArgsConstructor
public class DiscodeitException extends RuntimeException {

  private final Instant timestamp;
  private final ErrorCode errorCode;
  private final Map<String, Object> details;


}
```
과제 요구사항엔 details의 타입이 Map으로 정의되어 있어요~~

Reply:
이것도 다음 미션에 수정해서 반영하겠습니다!